trio/_tests/test_dtls.py

```diff
@@ -173,7 +170,7 @@ async def route_packet(packet: UDPPacket) -> None:
                         break
 
             def route_packet_wrapper(packet: UDPPacket) -> None:
-                try:
+                try:  # noqa: SIM105  # suppressible-exception
```

Comment:
loss of coverage information on whether a suppressed except is caught or not is actually a pretty good reason not to enable this check.